mu/interface/panes.py

```diff
@@ -597,16 +598,19 @@ def on_get(self, microbit_file):
         Fired when the get event is completed for the given filename.
         """
         msg = _(
-            "Successfully copied '{}' " "from the micro:bit to your computer."
+            "Successfully copied '{}' " "from the device to your computer."
         ).format(microbit_file)
         self.set_message.emit(msg)
         self.list_files.emit()
 
     def contextMenuEvent(self, event):
-        menu = QMenu(self)
+        menu_current_item = self.currentItem()
+        if menu_current_item is None:
+            return
         local_filename = self.currentItem().text()
```

Comment:
Could we re-use `menu_current_item` here..?

Reply:
Yes, thanks for the catch! I removed the fix temporarily to check the new tests failed and forgot to re-add that here.